vm/systemSmartContracts/auction.go

```diff
@@ -776,7 +773,7 @@ func (s *stakingAuctionSC) unStake(args *vmcommon.ContractCallInput) vmcommon.Re
 		return vmcommon.UserError
 	}
 	if len(args.Arguments) == 0 {
-		s.eei.AddReturnMessage("invalid number of arguments")
+		s.eei.AddReturnMessage(fmt.Sprintf("invalid number of arguments: expected min %d, got %d", 1, 0))
```

Comment:
You can use also above the same format: "expected min..." instead "min expected..."

Reply:
done.